nominal/experimental/stream_v2/_write_stream.py

```diff
@@ -58,17 +63,39 @@ def create(
             max_queue_size=batch_queue_maxsize,
         )
         batch_serialize_thread = spawn_batch_serialize_thread(
-            write_pool, clients, serializer, nominal_data_source_rid, batch_queue
+            write_pool, clients, serializer, nominal_data_source_rid, batch_queue, item_queue, track_metrics
         )
+
+        def add_metric_impl(channel_name: str, timestamp: IntegralNanosecondsUTC, value: float) -> None:
+            value = value / 1e9
```

Comment:
eh ok since there's no indication that `value` is a nanosecond timestamp, maybe it makes less sense to do the `/ 1e9` in here (at least if it's going to be named something like `add_metric`). Let's just put it at the call-site?